rules/windows/privilege_escalation_rogue_windir_environment_var.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/11/26"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/26"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a privilege escalation attempt via rogue Windows Directory (Windir) Environment Variable. This is a known
+primitive that is often combined with other vulnerabilities to elevate privileges.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Privilege Escalation via Windir Environment Variable"
+references = ["https://www.tiraniddo.dev/2017/05/exploiting-environment-variables-in.html"]
+risk_score = 71
+rule_id = "d563aaba-2e72-462b-8658-3e5ea22db3a6"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+type = "eql"
+
+query = '''
+registry where registry.path : ("HKEY_USERS\\*\\Environment\\windir", "HKEY_USERS\\*\\Environment\\systemroot") and 
```

Comment:
looks good, just waiting to have syntax validate properly in a dev tools instance before approving

Reply:
confirmed logic